<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="secLabel">
        <property name="text">
         <string>Время:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="secText">
        <property name="text">
         <string>0 : 0 : 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="circleLayout">
      <item>
       <widget class="QLabel" name="circleLabel">
        <property name="text">
         <string>Счётчик кругов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="circleText"/>
      </item>
      <item>
       <widget class="QPushButton" name="circleButton">
        <property name="text">
         <string>Круг</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="cleanButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>circleText</tabstop>
  <tabstop>circleButton</tabstop>
  <tabstop>cleanButton</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cleanButton</sender>
   <signal>clicked()</signal>
   <receiver>circleText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
